src/web/browser/ga/ga.ts

```diff
@@ -212,3 +212,24 @@ export const trackPerformance = (
         );
     }
 };
+
+export const trackVideosGA = ({
```

Comment:
Sorry to be picky, but the other functions in this file are called `trackNonClickInteraction` and `trackPerformance` so how about `trackVideoInteraction`?